install-deps.sh

```diff
@@ -239,6 +239,7 @@ function install_libpmem_on_ubuntu {
         $sha1 \
         $codename \
         check \
+        libndctl-dev \
```

Comment:
I exec `./install-deps.sh -DWITH_PMEM=1` and `./install-deps.sh WITH_PMEM=1`, but it didn't run here. So not be verified.

Reply:
`WITH_PMEM` is supposed to be an env variable, not a command line argument.

Reply:
`export WITH_PMEM=true`, then `install_deps.sh` the script will create `/etc/apt/sources.list.d/pmem.list` and failed.
```
E: Malformed line 1 in source list /etc/apt/sources.list.d/pmem.list (type)
E: The list of sources could not be read.
```
Still under study, never try it under env with "WITH_PMEM"

Reply:
@tchaikov detailed error msg in master branch.
https://gist.github.com/CongMinYin/beb61d8f9d7d90f4e60302853a89a349
After executing `install_deps.sh`. it will create a file named `/etc/apt/sources.list.d/pmem.list`, and met error.